api/src/test/resources/org/openmrs/module/fhir2/api/dao/impl/FhirObservationDaoImplTest_initial_data_suppl.xml

```diff
@@ -9,8 +9,8 @@
     graphic logo is a trademark of OpenMRS Inc.
 -->
 <dataset>
```

Comment:
Not sure why this is part of this commit?

Reply:
@ibacher Earlier I had changed this so that the tests pass for openmrsPlaformVersion 2.3.0. So I reverted it back to this after creating a new sub module